### PR TITLE
More feature tip at Configuration tab

### PR DESCRIPTION
**Pull request by HThuren**
Just add Help for some common used features

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: locales/en/messages.json [hunk 1]
```diff
@@ -1141,12 +1141,21 @@
     "featureSONAR": {
         "message": "Sonar"
     },
+    "featureSONARTip": {
+        "message": "Enable Sonar Ragefinder to measure distance to ground in cm"
```
Comment (haslinghuis) [inline locales/en/messages.json:1145]: Ragefinder :)

File: locales/en/messages.json [hunk 2]
```diff
@@ -1141,12 +1141,21 @@
...
+    },
     "featureTELEMETRY": {
         "message": "Telemetry output"
     },
+    "featureTELEMETRYTip": {
+        "message": "Enable Telemetry output to send to transmitter"
```
Comment (haslinghuis) [inline locales/en/messages.json:1151]: telemetry lower case

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: locales/en/messages.json (hunk 1 above)
Comment (HThuren) [reply locales/en/messages.json:1145]: Angry sonar ... :-)

Comment (haslinghuis) [reply locales/en/messages.json]: :rofl:

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]

**Pull request merged**